packages/maskbook/src/plugins/Trader/apis/trending/hotfix.ts

```diff
@@ -32,6 +35,11 @@ const KEYWORK_ID_MAP: {
         CRU: 'crust-network',
```

Comment:
@guanbinrui do we still need to hard code $CRU given cmc already have themhttps://coinmarketcap.com/currencies/crust/ and CRU is on the top list ? 

Note: gecko haven't updated crust $CRU on top list though.

cc @zhouhanseng